conans/test/integration/command/runner_test.py

```diff
@@ -0,0 +1,222 @@
+import textwrap
+import os
+import pytest
+import docker
+from conans.test.utils.tools import TestClient
+from conans.test.assets.cmake import gen_cmakelists
+from conans.test.assets.sources import gen_function_h, gen_function_cpp
+
+
+
+def docker_skip(test_image=None):
+    try:
+        docker_client = docker.from_env()
+        if test_image:
+            docker_client.images.get(test_image)
+    except docker.errors.DockerException:
+        return True
+    except docker.errors.ImageNotFound:
+        return True
+    except docker.errors.APIError:
+        return True
+    return False
+
+
+def conan_base_path():
+    import conans
+    return os.path.dirname(os.path.dirname(conans.__file__))
+
+
+def dockerfile_path(name=None):
+    path = os.path.join(os.path.dirname(__file__), "dockerfiles")
+    if name:
+        path = os.path.join(path, name)
+    return path
+
+
+@pytest.mark.skipif(docker_skip(), reason="Only docker running")
+@pytest.mark.xfail(reason="conan inside docker optional test")
+def test_create_docker_runner_dockerfile_folder_path():
+    """
+    Tests the ``conan create . ``
+    """
+    client = TestClient()
+    profile_build = textwrap.dedent(f"""\
+    [settings]
+    arch=x86_64
+    build_type=Release
+    compiler=gcc
+    compiler.cppstd=gnu17
+    compiler.libcxx=libstdc++11
+    compiler.version=11
+    os=Linux
+    """)
+    profile_host = textwrap.dedent(f"""\
+    [settings]
+    arch=x86_64
+    build_type=Release
+    compiler=gcc
+    compiler.cppstd=gnu17
+    compiler.libcxx=libstdc++11
+    compiler.version=11
+    os=Linux
+    [runner]
+    type=docker
+    dockerfile={dockerfile_path()}
+    docker_build_context={conan_base_path()}
+    image=conan-runner-default-test
+    cache=copy
+    remove=True
+    """)
+    conanfile = textwrap.dedent("""
+        from conan import ConanFile
+
+        class MyTest(ConanFile):
+            name = "pkg"
+            version = "0.2"
+            settings = "build_type", "compiler"
+            author = "John Doe"
+            license = "MIT"
+            url = "https://foo.bar.baz"
+            homepage = "https://foo.bar.site"
+            topics = "foo", "bar", "qux"
+            provides = "libjpeg", "libjpg"
+            deprecated = "other-pkg"
```

Comment:
Most of the metadata, unless used in the test, can be removed to simplify the recipe.